nginx/default.template.conf

```diff
@@ -17,6 +17,10 @@ server {
   root /usr/share/nginx/html;
   index index.html index.html;
 
+  location /api/(.*) {
+    proxy_pass ${API_URL}/$1;
```

Comment:
suggestion: in a follow up PR we could add some docs to make it clear that this env var is now to be defined as the URL from the perspective of the server hosting the UI and not the client accessing it